project/ws/app/src/lib/routes/app-toc/components/app-toc-banner/app-toc-banner.component.html

```diff
@@ -454,18 +454,19 @@ <h2 class="mat-headline margin-remove-bottom ws-mat-black-text">{{'contentSharin
                                 </div>
                             </mat-option>
                         </mat-autocomplete>
-                        <mat-hint align="start">{{'contentSharing.note' | translate}}</mat-hint>
-                        <mat-hint align="end" class="left-over-emails">{{ users.length }} /{{maxEmailsLimit}} {{'contentSharing.remaingEmails' | translate}}</mat-hint>
+                        <mat-hint align="start"><b>{{'contentSharing.note' | translate}}</b></mat-hint>
```

Comment:
can you please check if we can use class to make font bold instead using <b> tag